.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,7 @@
+### Expected results
```

Comment:
I think Actual results should come before Expected, since that will describe the issue.